metacat-connector-polaris/src/main/java/com/netflix/metacat/connector/polaris/PolarisDatabaseEntity.java

```diff
@@ -0,0 +1,44 @@
+package com.netflix.metacat.connector.polaris;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Version;
+
+/**
+ * Entity class for Database object.
+ */
+@Getter
+@Setter
```

Comment:
Why have all fields expose setter? Forcing setting of most required fields at construction time and preferring immutable objects when possible should be the preferred pattern.

Reply:
Using the jpa option of field based instead of property based access makes the setters not needed. Making objects more immutable where possible is a better pattern to follow, for example some fields like generated primary key should purely be managed by the persistence provider and never be set by the application so making the fields private and not exposing any way to set them better ensures against incorrect access.

Reply:
my question was simply if jpa framework allowed it. 
i tried it and it works. 
PR updated.